app/utilities/schema.py

```diff
@@ -22,6 +24,18 @@
 
 LANGUAGES_MAP = {"test_language": [["en", "cy"]]}
 
+SCHEMA_REQUEST_MAX_BACKOFF = 0.2
+SCHEMA_REQUEST_MAX_RETRIES = 2  # Totals no. of request should be 3.
```

Comment:
Worth expanding. `# Totals no. of request sshould be 3. The initial request + SCHEMA_REQUEST_MAX_RETRIES`

Reply:
Done 👍